packages/shared/src/components/CombatReport/CombatReplay/ReplayUnitFrames/UnitFrame.tsx

```diff
@@ -213,10 +213,18 @@ export const UnitFrame = (props: IProps) => {
     }
     if (sd['2050']) {
       // 235587 Miracle Worker (Holy Word: Serenity)
-      if (props.unit.info?.pvpTalents.includes('235587')) {
+      if (props.unit.info?.talents.find((t) => t?.id2 === 103737)) {
```

Comment:
is this line change intentional? doesn't line up with the comment above

Reply:
Yeah. I will fix the comment. Some of this stuff never got corrected for DF style talents...